network/web/browser/google-chrome-stable/pspec.xml

```diff
@@ -41,6 +41,14 @@
     </Package>
 
     <History>
+        <Update release="119">
+            <Date>05-10-2018</Date>
```

Comment:
Should be `05-14-2018` seeing as that's what today is. The date is not meant for when the release happened but rather when the package or pspec update occurred.

Reply:
Oh sorry, will fix